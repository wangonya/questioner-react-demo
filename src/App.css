@import url("https://fonts.googleapis.com/css?family=Roboto");

* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}

html {
  color: #212121;
  font-size: 1em;
  line-height: 1.4;
  min-height: 100%;
  position: relative;
}

body {
  font-family: "Roboto", sans-serif;
  margin: 0 0 70px;
}

.container {
  padding: 30px;
  display: grid;
}

a {
  color: #53e3a6;
  text-decoration: none;
  transition: color 0.3s ease;
}

a:hover {
  color: #f0776c;
}
