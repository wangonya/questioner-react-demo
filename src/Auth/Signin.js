import React, { Component } from "react";

class Signin extends Component {
  render() {
    return <div>Signin Page</div>;
  }
}

export default Signin;
