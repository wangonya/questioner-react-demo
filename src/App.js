import React, { Component } from "react";
import { BrowserRouter as Router, Route, Link } from "react-router-dom";
import { Provider } from "react-redux";

import "./App.css";

import Navbar from "./Navbar/Navbar.js";
import store from "./redux/store";

class App extends Component {
  render() {
    return (
      <Provider store={store}>
        <Navbar />
      </Provider>
    );
  }
}

export default App;
