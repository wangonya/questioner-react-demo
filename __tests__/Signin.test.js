import React from "react";
import { shallow } from "enzyme";
import { Provider } from "react-redux";

import Signin from ".././src/Auth/Signin";
import { storeFactory } from ".././src/testUtils";

const setUp = (initialState = {}) => {
  const store = storeFactory(initialState);
  const wrapper = shallow(<Signin store={store} />).dive();
};

describe("<Signin /> component", () => {
  let wrapper;
  beforeEach(() => {
    const initialState = {};
    wrapper = setUp(initialState);
  });
  test("renders without crashing", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
