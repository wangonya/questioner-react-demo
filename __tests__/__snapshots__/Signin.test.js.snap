// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Signin /> component renders without crashing 1`] = `ShallowWrapper {}`;
